File: CMS/src/app/shared/DanhMucXepLoai.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucXepLoai } from 'src/app/shared/DanhMucXepLoai.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucXepLoaiService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucXepLoai";
    }
}

File: Service/Implement/DanhMucXepLoaiService.cs
namespace Service.Implement
{
    public class DanhMucXepLoaiService : BaseService<DanhMucXepLoai, IDanhMucXepLoaiRepository>
    , IDanhMucXepLoaiService
    {
    private readonly IDanhMucXepLoaiRepository _DanhMucXepLoaiRepository;
    public DanhMucXepLoaiService(IDanhMucXepLoaiRepository DanhMucXepLoaiRepository) : base(DanhMucXepLoaiRepository)
    {
    _DanhMucXepLoaiRepository = DanhMucXepLoaiRepository;
    }
    }
    }

